website/api/zkvm/zkvm_overview.md

```diff
@@ -65,6 +65,15 @@ Once you've built a zkVM application, you can generate proofs on your own machin
 For local proof generation, we recommend a machine with at least 16GB of RAM.
 Options for generating proofs using GPU acceleration and skipping proof generation are documented in the [feature flags].
 
+## Key Features
+
+There are several key features that distinguish RISC Zero's zkVM from others. These features enable developers to more easily develop zk powerful applications.
+
+### Continuations for limitless computations
+
+Continuations allow the zkVM to do large computations by splitting the work into small, independent segments of the same size.
```

Comment:
Just saw your comment on concepts in https://github.com/risc0/risc0/issues/1071#issuecomment-1793161092. If you wanted, you could link to the terminology page here.
```suggestion
Continuations allow the [zkVM] to do large computations by splitting the work into small, independent [segments] of the same size.
```
You would need to add `[zkVM]: /terminology#zero-knowledge-virtual-machine-zkvm` and `[segments]: /terminology#segment` to the end of the article too if you do this.